portfolio/src/main/webapp/script/comment-script.js

```diff
@@ -12,12 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// TODO: Modify script to append n number of comments from servlet.
+
 /**
  * Fetches data from servlet and sets it in the comments section of portfolio.
  * Called whenever comments section is loaded.
+ * TODO: Modify to handle dynamic data.
  */
 async function getData() {
   const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('data-container').innerText = quote;
+  const jsonData = await response.json();
+  
+  const commentsListElement = document.getElementById('comments-container');
+  commentsListElement.innerHTML = '';
+  for(comment of jsonData){
```

Comment:
```suggestion
  for (comment of jsonData) {
```

Reply:
Done, spacing issues in `DataServlet.java` also fixed.